### PR TITLE
Create Python Entry for random.sample()

### PR DESCRIPTION
**Pull request by SSwiniarski**
### Description

Add entry for python/concepts/random-module/terms/sample

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.

### CODE REVIEW

**Comment by SSwiniarski** [comment]
Another orphan from original assignment sheet.

File: content/python/concepts/random-module/terms/sample/sample.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  - 'Random'
+  - 'Functions'
+  - 'Statistics'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The `random.sample()` method takes a population (list, range, set) and a number as arguments and returns a list consisting of that number of randomly selected, non-duplicated, items from the population.
```
Comment (KTom101) [inline content/python/concepts/random-module/terms/sample/sample.md:17]:
```suggestion
The `random.sample()` method takes a population (list, range, set, or string) and a number as arguments. It returns a list with the specified number of items. Each item is randomly selected without duplication from the population.
```

File: content/python/concepts/random-module/terms/sample/sample.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+## Syntax
+
+```pseudo
+random.sample(<population>,<number>)
+```
+
+Where `<population>` is the list (or range, or set) where items are being selected from, and `<number>` is the number of items selected to return.
```
Comment (KTom101) [inline content/python/concepts/random-module/terms/sample/sample.md:25]:
```suggestion
Where `<population>` is the sequence the items are selected from, and `<number>` is the number of item(s) to be returned.
```

**Review by KTom101 — COMMENTED** [review]
@SSwiniarski Just completed my review for this entry.

**Comment by SSwiniarski** [comment]
@KTom101, I implemented your suggestions.

File: content/python/concepts/random-module/terms/sample/sample.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+```pseudo
+random.sample(<population>,<number>)
+```
+
+Where `<population>` is the sequence the items are selected from, and `<number>` is the number of item(s) to be returned.
+
+## Codebyte Example
+
+```codebyte/py
```
Comment (Dusch4593) [inline content/python/concepts/random-module/terms/sample/sample.md:29]:
```suggestion
```codebyte/python
```

According to the `LanguageOptions` constants for codebytes in [Codecademy's `client-modules` repo](https://github.com/Codecademy/client-modules/blob/main/packages/codebytes/src/consts.ts) should be spelled out. This is also the case when commenting on [the discuss forums](https://discuss.codecademy.com/)

File: content/python/concepts/random-module/terms/sample/sample.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+---
+Title: '.sample()'
+Description: 'Returns a given number of randomly selected items from a given population.'
```
Comment (Dusch4593) [inline content/python/concepts/random-module/terms/sample/sample.md:3]:
```suggestion
Description: 'Returns a list of items randomly selected from a given population.'
```

File: content/python/concepts/random-module/terms/sample/sample.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+The `random.sample()` method takes a population (list, range, set, or string) and a number as arguments. It returns a list with the specified number of items. Each item is randomly selected without duplication from the population.
+
+## Syntax
+
+```pseudo
+random.sample(<population>,<number>)
+```
+
+Where `<population>` is the sequence the items are selected from, and `<number>` is the number of item(s) to be returned.
```
Comment (Dusch4593) [inline content/python/concepts/random-module/terms/sample/sample.md:25]:
```suggestion
The `population` is a sequence of items can either be a [list](https://www.codecademy.com/resources/docs/python/lists), [string](https://www.codecademy.com/resources/docs/python/strings), or [range](https://www.codecademy.com/resources/docs/python/built-in-functions/range). If [sets](https://www.codecademy.com/resources/docs/python/sets) are used, they should be converted to a list or [tuple](https://www.codecademy.com/resources/docs/python/tuples) before being passed to this method.
```

Looks like [newer versions of `random.sample()` aren't so nice to sets](https://docs.python.org/3/library/random.html#:~:text=Deprecated%20since%20version%203.9%3A%20In%20the%20future%2C%20the%20population%20must%20be%20a%20sequence.%20Instances%20of%20set%20are%20no%20longer%20supported.%20The%20set%20must%20first%20be%20converted%20to%20a%20list%20or%20tuple%2C%20preferably%20in%20a%20deterministic%20order%20so%20that%20the%20sample%20is%20reproducible.). Let's err on the side of caution and use the recommendations described in the suggestion above.

File: content/python/concepts/random-module/terms/sample/sample.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+## Codebyte Example
+
+```codebyte/py
+import random
+
+myRange = range(1000)
+
+mySample = random.sample(myRange,10)
+
+print(mySample)
```
Comment (Dusch4593) [inline content/python/concepts/random-module/terms/sample/sample.md:36]:
```suggestion
my_range = range(1000)

my_sample = random.sample(my_range,10)

print(my_sample)
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@SSwiniarski Done with my review and left some comments/suggestions. 😄

**Comment by SSwiniarski** [comment]
@Dusch4593, I committed your suggestion and rewrote the one sentence to avoid the potential misunderstanding you mentioned about selection affecting subsequent samples.

**Review by Dusch4593 — APPROVED** [review]
@SSwiniarski LGTM!

**Comment by Dusch4593** [comment]
@yangc95 @KTom101 anything left to check or approve before we merge this one?

**Review by yangc95 — APPROVED** [review]
LGTM 👌🏻

**Comment by Dusch4593** [comment]
@SSwiniarski Another solid entry! 💪 Live url here: https://www.codecademy.com/resources/docs/python/random-module/sample

**Pull request merged**